test/intern.js

```diff
@@ -0,0 +1,35 @@
+
+define({
+  proxyPort: 9000,
+  proxyUrl: "http://localhost:9000/",
+  capabilities: {},
+  environments: [
+    { browserName: "chrome" },
+    { browserName: "internet explorer", version: [ "11", "10" ] },
+    { browserName: "firefox", version: "34" }
```

Comment:
this should include all the supported browsers see https://github.com/jquery/globalize/blob/master/browserstack.json